content/_data/changelogs/weekly.yml

```diff
@@ -982,6 +982,29 @@
       pull: 2956
     # pull: 2971 not notable enough
     # pull: 2968 not notable enough
+- version: "2.75"
+  date: 2017-08-20
+  changes:
+    - type: rfe
+      message: >
+        Update Windows service wrapper from 2.1.0 to 2.1.2 and Windows Agent Installer from 1.9 to 1.9.1.
+      references:
+        - url: https://github.com/kohsuke/winsw/blob/master/CHANGELOG.md
+          title: Windows service wrapper changelog
+        - url: https://github.com/jenkinsci/windows-slave-installer-module/blob/master/CHANGELOG.md
+          title: Windows agent installer changelog
+        - pull: 2987
+    - type: rfe
+      message: >
+         Disable obsolete JNLP, JNLP2, and CLI protocols on new Jenkins installations during Setup Wizard.
+      issue: 45841
+      pull: 2950
```

Comment:
Add link to the announcement blogpost: https://jenkins.io/blog/2017/08/11/remoting-update/